spec/apollo-federation/service_field_v2_spec.rb

```diff
@@ -128,6 +128,98 @@ def execute_sdl(schema)
       )
     end
 
+    it 'returns the federation SDL with multiple links for the schema' do
+      product = Class.new(base_object) do
+        graphql_name 'Product'
+
+        field :upc, String, null: false
+      end
+
+      query_obj = Class.new(base_object) do
+        graphql_name 'Query'
+
+        field :product, product, null: true
+      end
+
+      schema = Class.new(base_schema) do
+        query query_obj
+        federation version: '2.3',
+                   links: [{
```

Comment:
Can you explain the need to support multiple links in this PR? I wonder if we can do that separately and skip that for this PR, to keep it smaller.

Reply:
Per the [composeDirective docs](https://www.apollographql.com/docs/federation/federated-types/federated-directives/#composedirective):
> The directive to preserve must be defined and imported from a core specification via the @link directive. 

I think `composeDirective` would be unusable without support for multiple links

Reply:
Also if we declare a custom directive as part of the https://specs.apollo.dev link it fails. For example for:
```
 @link(
    url: "https://specs.apollo.dev/federation/v2.3"
    import: ["@shareable", "@key", "@composeDirective", "@myCustomDirective"]
  )
```
compose fails with:
```
UNKNOWN: [...] Cannot import unknown element "@myCustomDirective".
```

Reply:
Got it. Makes sense.